definitions/npm/bizniz_v1.x.x/test_bizniz_v1.x.x.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import bizniz from "bizniz";
+
+(bizniz.addWeekDays(new Date(0), 1): Date);
```

Comment:
Use `describe` and `it` blocks please. :)

Reply:
done.